scipy/stats/_stats_py.py

```diff
@@ -7617,11 +7628,22 @@ def ks_1samp(x, cdf, args=(), alternative='two-sided', method='auto'):
 
     Returns
     -------
-    statistic : float
-        KS test statistic, either D, D+ or D- (depending on the value
-        of 'alternative')
-    pvalue : float
-        One-tailed or two-tailed p-value.
+    res: KstestResult
+        An object containing attributes:
+
+        statistic : float
+            KS test statistic, either D, D+ or D- (depending on the value
+            of 'alternative')
+        pvalue : float
+            One-tailed or two-tailed p-value.
+        statistic_location: float
+            Value of the stochastic variable at the KS statistic. By
+            definition, the distance between the theoretical and sampled
+            distributions is largest at this location.
+        statistic_sign: str
+            1 if the KS statistic is the maximal positive difference between
+            empirical and theoretical distribution, -1 if the KS statistic is
+            the negative difference.
```

Comment:
```suggestion
            1 if the KS statistic is the maximal positive difference between
            empirical and hypothesized distributions (D+), -1 if the KS
            statistic is the negative difference (D-).
```
Because "D" is ultimately either D+ or D-, do you think we should alter the description above to eliminate "D"?

> KS test statistic, either D+ or D- (depending on the value of 'alternative')